src/operation/overlayng/RingClipper.cpp

```diff
@@ -120,20 +120,24 @@ RingClipper::intersectionLineX(const Coordinate& a, const Coordinate& b, double
 bool
 RingClipper::isInsideEdge(const Coordinate& p, int edgeIndex) const
 {
+    if (clipEnv.isNull()) {
```

Comment:
@dr-jts Another one here.

Reply:
Technically yes, but OverlayNG should never call `RingClipper` with an empty clip envelope.

Reply:
Well, the OverlayNG cases are both hit by the test suite -- that's why I added these guards.

Reply:
Ah right, the unit tests cover more cases than should be seen in practice.  Good to have the guards, anyway.